src/main/java/by/custom/utilcalculator/exception/TreeReadingException.java

```diff
@@ -0,0 +1,12 @@
+package by.custom.utilcalculator.exception;
+
+import by.custom.utilcalculator.exception.constants.UtilsborErrorCode;
+import by.custom.utilcalculator.exception.constants.UtilsborErrorDescription;
+
+import java.io.IOException;
+
+public class TreeReadingException extends IOException {
```

Comment:
This name gives more information: what tree, is it our service exception or pure java exception?
```suggestion
public class UrilsborCommandTreeReadingException extends IOException {
```

Reply:
Renamed, thanks